pkg/gds/emails/emails_test.go

```diff
@@ -48,31 +48,43 @@ func TestVerifyContactURL(t *testing.T) {
 		Token: "1234defg4321",
 		VID:   "42",
 	}
-	link, err := url.Parse(data.VerifyContactURL())
-	require.NoError(t, err)
-	require.Equal(t, "https", link.Scheme)
-	require.Equal(t, "vaspdirectory.net", link.Host)
-	require.Equal(t, "/verify-contact", link.Path)
-	params := link.Query()
-	require.Equal(t, data.Token, params.Get("token"))
-	require.Equal(t, data.VID, params.Get("vaspID"))
+	require.Empty(t, data.VerifyContactURL(), "if no base url is provided, VerifyContactURL() should return empty string")
```

Comment:
I didn't realize you could add a debug string to the require, that's pretty useful!